assets/sass/cds/_base.scss

```diff
@@ -221,3 +221,7 @@ input.error {
 .error-message {
 	color: red;
 }
+
+ol ul, ol ol, ul ul, ul ol {
```

Comment:
Here's a change to remove the whitespace that was showing up between bullets 1 and 2 in the posting (and elsewhere on the site where this might have been occurring).